src/main/java/utils/InputParser.java

```diff
@@ -0,0 +1,56 @@
+package utils;
+
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class InputParser {
+    private static Pattern intPattern = Pattern.compile("\\d+");
+
+    public static boolean IsInteger(String s) {
+        return intPattern.matcher(s).matches();
+    }
+
+    private static Pattern flagPattern = Pattern.compile("/\\w+");
+    /**
```

Comment:
Maybe a blank line could be added between the variable and the javadoc for the method below to separate logical units